apps/docs/src/pages/index.mdx

```diff
@@ -14,21 +14,21 @@ Assuming that a question has been solved in your server and no where else on the
 | 2. Find the question indexed from your server in Google          | 2. No relevant results found                                                                                               |
 | 3. Read the answer to the question                               | 3. Try to find a place to ask the question                                                                                 |
 | 4. Join the server linked on the question for further discussion | 4. Find a Discord server and join it (if the invite link is still valid)                                                   |
-|                                                                  | 5. Search in the server I just joined for the question using the small Discord search sidebar                              |
-|                                                                  | 6. (Maybe) Find the question I'm looking for                                                                               |
-|                                                                  | 7. If I don't find the question or don't search for it, I ask my question in the server, resulting in a duplicate question |
+|                                                                  | 5. Search in the server you just joined for your question
+                        |
+|                                                                  | 6. May or may not get results                                                                                             |
+|                                                                  
 
-Answer Overflow allows for people to focus on building with your product rather than hunting down answers to questions that have already been answered.
+Answer Overflow allows for you to focus on building your product rather than searching for answers to questions that have been answered
 
-**Your community is amazing and Answer Overflow is a way to share it with even more
-people, allowing all of the work that people have put into providing support to be
-even more valuable.**
+**It is also is a easy way to showcase your community’s resources via indexing on Google
```

Comment:
Can we rephrase this to make it a bit more actiony / dynamic? At the moment the sentence is a bit flat but the goal is to try to make them excited about adding AO